.github/workflows/scripts/biome_disable_check.py

```
@@ -86,14 +84,14 @@
                 for file_name in files:
                     if file_name.endswith(".ts") :
                         file_path = os.path.join(root, file_name)
-                        if has_eslint_disable(file_path):
+                        if has_biome_ignore(file_path):
                             print(
-                                f"""File {file_path} contains eslint-disable
+                                f"""File {file_path} contains biome-ignore
                                 statement."""
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)